odpf/optimus/core/v1beta1/job_spec.proto

```diff
@@ -90,10 +90,18 @@ message DeployJobSpecificationResponse {
 
   // non ack responses are more of a progress/info response
   // and not really success or failure statuses
-  bool ack = 2;
+  // this is deprecated and being replaced with type field
+  bool ack = 2 [deprecated=true];
```

Comment:
want to discuss on these contracts `type` and `ack`

Reply:
@sravankorumilli @deryrahman  just want to confirm, is this resolved? As the PR in Optimus has merged.